.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,3 +8,20 @@ Follow this checklist to help us incorporate your contribution quickly and easil
  - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).
 
 It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.
+
+------------------------------------------------------------------------------------------------
+PostCommitName|PostCommitStatus
```

Comment:
Could this table be shortened to something like:

**Postcommit Status**
Go [![Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/badge/icon)](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/)
Java [![Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/badge/icon)](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) [![Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/badge/icon)](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) [![Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/badge/icon)](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) [![Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/badge/icon)](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) [![Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/badge/icon)](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) [![Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/badge/icon)](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
...

I don't like the PR description to take too much vertical space.

Reply:
The `ValidatesRunner` tests are more usefully cataloged as tests of the runner. The suites happen to be implemented in Java but that isn't what is under test.

How about?

### Health of `master`

| **SDK**       | Go  | Python | Java |
|------------|------|------|-----|
| Status    | [ ![Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/badge/icon) ](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | [ ![Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/badge/icon) ](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | [ ![Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/badge/icon) ](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) |

| **Runner** | Apex | Dataflow | ... | 
|------------|------|------|-----|
| Java ValidatesRunner | [ ![Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/badge/icon)](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | ... | ... |
| Py ValidatesRuner | n/a | [![Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/badge/icon)](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/)  | ... |

And what does ValCont even mean?